content/blog/technical-assessment-evaluation-criteria.md

```diff
@@ -0,0 +1,104 @@
+2. Self-paced, prerequisite learning
+3. Call with the People Success Team
+4. Technical Discussion
+5. Hands-on Pairing
+
+## Self-paced, Prerequisite Learning
+Once your application is shortlisted, you will receive an email with a list of resources and short videos. This email is meant to help prepare you for the interview process and set you up for success. Suggested learning in the email is self-paced and there is no time limit for you to complete it - we're perpetually hiring!
+
+Some of our favourites: 
+- [Test-Driven Development](https://www.youtube.com/watch?v=qkblc5WRn-U)
+- [Software Craftsmanship](https://www.youtube.com/watch?v=c07uQGBZl0A)
+- [Clean Code](https://www.youtube.com/watch?v=zV079g7Irks&feature=emb_imp_woyt)
+
+The email also highlights other skills needed for the interviews. For example, knowing common Git operations, hands-on knowledge of your IDE (with shortcuts), and an understanding of a unit testing framework to name a few!
+
+## Call with our People Success Team
+This step helps align your expectations from the role with what we have to offer. Being remote-first also means that it is important we're aligned not just on tech; but also on culture, attitude, and core values! Our People Success Team would also love to answer any questions you have about the Company, so ask away!
+
+## Technical Discussion with an Incubyte Craftsperson
+A 30-minute discussion focused on your technical background and journey so far. You can expect the conversation to revolve around languages, frameworks, and databases you have worked with; your prior experiences, projects, etc. This is for us to get introduced to the depth and breadth of your technical expertise. Here it is important that you have a good Wi-Fi connection since we'd want to have your video on during the discussion.
+
+## Hands-on Pairing with an Incubyte Craftsperson
+This round is where the fun begins! Fire up your IDE with the language of your choice, a simple Hello World program pre-installed, and get ready to get your hands dirty. We will solve a short Kata together as a team! Consider this round as a pairing exercise, in which an Incubyte Craftsperson will partner with you and act as a navigator. It is fine to look stuff up on the internet. We understand that having Maven or Gradle dependency coordinate on top of the code is difficult. (We Google things while working on production projects ALL the time - its the thought process that matters!)
+
+This brings us to the end of our interview process. Expect to walk away with an offer once you have cleared all these rounds. We've even put together some additional information to help you crack it! You're welcome. ;)
+
+The entire process can be broken down into 2 parts:
+1. Technical
+2. Non-technical  
+
+## Technical
+At this point we're trying to get to know your technical self better. How you code, think and generally vibe with the things that matter to us will be taken into consideration here. Let's start:
```

Comment:
We want to see how you code, think and generally vibe with the things that matter to us. Let's start: